### PR TITLE
Started the draft for release notes

### PR DESCRIPTION
**Pull request by gvbalaji**
Started the draft for release notes

### CODE REVIEW

File: release-notes/release-notes.md [hunk 1]
```diff
@@ -1,26 +1,63 @@
...
+* Auto-scale up of UPF, SMF and AMF services based on changes to capacity requirements expressed as user intent.
 
 ## Limitations
+* In terms of infrastructure automation, only creation of KIND clusters is supported.
+* Deployment of  free5gc functions other than SMF, UPF and AMF may need some manual configuration such as IP addresses.
+* Inter cluster networking is not dynamic which means as more clusters are deployed some manual tweak will be needed for inter cluster communications. 
+* Feedback of workload deployments from workload clusters to the management cluster is limited. You may need to directly connect to the workload cluster using kubectl to debug the deployment issues. 
+* Web UI features are limited to view/edit of Package Variants and Package variant sets. More features will be added in subsequent releases. 
+* When the capacity of UPF,SMF and AMF is changed, the free5gc operator on the workload cluster will instantiate a new POD with correspondingly modified resources (CPU, memory etc.) During this pods will restart. This is the limitation of free5gc.
```
Comment (debeaueric) [inline release-notes/release-notes.md:57]: Security: WebUI with no IAM,

File: release-notes/release-notes.md [hunk 2]
```diff
@@ -1,26 +1,63 @@
...
+    * Vsphere Version ?
+    * Openstack version ?
+    * Vagrant on Virtual box on Windows 10/11.
 * Supported cloud environments
+  * Google Cloud Platform
+* **Other k8s systems?**
 
 ## Features
+### API
+CRDs provided for UPF, SMF and AMF 5g core services
```
Comment (debeaueric) [inline release-notes/release-notes.md:35]: and more: cf https://github.com/nephio-project/api/tree/main

File: release-notes/release-notes.md [hunk 3]
```diff
@@ -1,26 +1,63 @@
...
+* Kpt packages for all [free5gc](https://free5gc.org/) services
+* Packages for core Nephio services
+* Packages for cluster API services for cluster creation
+* Packages for dependent services
+### Functionalities
+* Create kubernetes clusters. This functionality ia based on cluster API. At this time only KIND clusters creation is supported.
+* Fully automated deployment of UPF, SMF and AMF services of [free5Gc](https://free5gc.org/) . These are deployed on multiple clusters based on user's intent expressed via CRDs.
+* Inter cluster networking setup.
+* Deployment of other free5gc functions. Some manual configuration such as IP addresses may be needed for these services.
+* Auto-scale up of UPF, SMF and AMF services based on changes to capacity requirements expressed as user intent.
```
Comment (debeaueric) [inline release-notes/release-notes.md:49]: Package management

File: release-notes/release-notes.md (hunk 1 above)
Comment (debeaueric) [inline release-notes/release-notes.md:57]: Repos must be based on Github or Gitea

File: release-notes/release-notes.md [hunk 4]
```diff
@@ -1,26 +1,63 @@
...
+
+The solution is intended to address the initial provisioning of the network functions and the underlying cloud infrastructure, and also provide Kubernetes-enabled reconciliation to ensure the network stays up through failures, scaling events, and changes to the distributed cloud. 
+
+Nephio leverages "configuration as data" principle and Kubernetes declarative, actively-reconciled methodology along with machine-manipulable configuration to tame the complexity of Network Functions deployment and life cycle management..
+
+This release of Nephio focuses on:
+* Exhibiting the core Nephio principles such as Configuration as data and leveraging the intent driver, actively reconciled nature of kubernetes.
+*  Infrastructure orchestration/automation using controllers based on  cluster API. At this time only KIND cluster creation is supported.
+* Orchestration/automation of 5G core network functions deployment and management. This release focuses on network functions from free5gc.
```
Comment (debeaueric) [inline release-notes/release-notes.md:16]: Also explain that NFDeployment concept is introduced and may be applied for any NF

**Comment by johnbelamaric** [comment]
I am going to merge all these and we will iterate from here.

/approve
/lgtm

**Pull request merged**